Civi/Api4/Utils/CoreUtil.php

```diff
@@ -73,4 +73,34 @@ public static function getApiNameFromTableName($tableName) {
     return $entityName;
   }
 
+  /**
+   * Given an expression like "activity_date_time IS previous.month", rewrite to
+   * an equivalent expression "activity_date_time BETWEEN {start-of-previous-month} AND {end-of-previous-month}.
+   *
+   * @param string $fieldName
+   *   Ex: 'activity_date_time'
+   * @param string $operator
+   *   Ex: 'IS' or 'IS NOT'
+   * @param string $criteria
+   *   Ex: ['IS' => 'previous.month']
+   * @return array|NULL
+   *   Array(string $newOperator, mixed $newCriteria).
+   *   Ex: ['BETWEEN' => ['2020-05-01', '2020-06-01']]
+   */
+  public static function rewriteIsCriteria($fieldName, $operator, $criteria) {
```

Comment:
Would this function be better situated in `\Civi\Api4\Utils\FormattingUtil`?

Reply:
Yeah, I wasn't really sure. `FormattingUtil` seems just as well.